terraform/README.md

```diff
@@ -1,3 +1,45 @@
+<em><h1>Wordpress Project - Group 2</h1></em>
+Here we will discuss how to run the Project solution
+
+<em><h2>How to run</h2><p></em>
+- Take a clone or fork of the repoistry from Git
+<br>
+- Navgiate to the terraform folder (cd terraform)
+<br>
+- You'll need to initialize terraform to do this, please type ```terraform init```
+<br>
+- Once completed, type ```terraform plan``` This will plan out the way the Infrastructure is deployed, if you recieve any issues    during this phase, please inform the team.
+<br>
+- Finally finish off with ```terraform apply``` and hit ```yes``` when prompoted. (You may recieve some issues regarding a lock file when applying, if this is the case please add the following parameter -lock=false)
+<br>
+- Once review has been completed, please don't forget to type ```terraform destroy -lock=false``` (Note there may be some issues regarding a DB Snapshot when destroying the code, if you do recieve this error navigate to your AWS RDS Console and select Snapshots from the left hand side, and delete the snapshot related to the created DB cluster.</p>
```

Comment:
I wouldn't recommend using the -lock=false.. thic could corrupt your state file and is not a good practice.
If the state is locked, is mostly because you're running two terraform commands at the same time or breaking the terraform command execution.

Reply:
Noted,
Thanks Kiko.
Looks like I may have an issue with mine.

With the RDS, there was always issues when deleting it after it was created in Terraform..
Either subnets would stay, or the DB won't get deleted because of the snapshots which lead to a lot of issues initially.
I ended up going to the AWS Console for troubleshooting.

Is this expected ?